erc/ERCS/erc-7562.md

```diff
@@ -249,44 +340,48 @@ That is, if the `validateUserOp()` is rejected for any reason, it is treated as
 
 * Definitions:
     * **`opsSeen`, `opsIncluded`, and reputation calculation** are defined above.
-    * `UnstakedReputation` of an entity is a maximum number of entries using this entity allowed in the mempool.
-    * `opsAllowed` is a reputation based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
-* **[UREP-010]** `UserOperation` with unstaked sender are only allowed up to `SAME_SENDER_MEMPOOL_COUNT` times in the mempool.
+    * `UnstakedReputation` of an entity is the maximum number of entries using this entity allowed in the mempool.
+    * `opsAllowed` is a reputation-based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
+* **[UREP-010]** `UserOperation` with unstaked sender is only allowed up to `SAME_SENDER_MEMPOOL_COUNT` times in the mempool.
 * **[UREP-020]** For other entities: \
-    `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
-    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for new entity
+  `opsAllowed = SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT + (inclusionRate * INCLUSION_RATE_FACTOR) + (min(opsIncluded, 10000)`.
+    * This is a default of `SAME_UNSTAKED_ENTITY_MEMPOOL_COUNT` for a new entity
 * **[UREP-030]** If an unstaked entity causes an invalidation of a bundle, its `opsSeen` is set to `1000`, effectively blocking it from inclusion for 24 hours.
```

Comment:
Not reflected by the above calculation.  Changing `opsSeen` affects `inclusionRate` (unless if it already has more than 1000 opsIncluded, in which case `inclusionRate` actually becomes bigger than 1).  But then `inclusionRate` itself doesn't really affect `opsAllowed` in the above calculation.

Also, this rule should be consistent `SREP-050` (which specifies opsSeen increment by 10000, and also needs to be fixed).

I think both UREP-030 and SREP-050 should **set** opsSeen to a high number (whether 1000 or 10000 should be calculated to achieve the desired duration) and opsIncluded to 0.

Reply:
I see SREP-050 was fixed now. Shouldn't UREP-030 work the same way?

(and the calculation above still seems wrong)